packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```diff
@@ -37,6 +37,7 @@
         "stopped": "stopped",
         "failed": "Inject failed: __error__",
         "label": {
+            "properties": "Properties",
```

Comment:
Went with `Properties` to describe the form-row where the user manages the msg properties. The bad thing is that the same word is displayed in the config node's tab: 
![image](https://user-images.githubusercontent.com/4663918/72996774-4004d200-3dc9-11ea-91f4-5c1aab68017e.png)

Reply:
I think it would look fine without that duplicate label - and would save translating it also.

Reply:
Good idea. Done.